<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadReportFromSQL</class>
 <widget class="QDialog" name="LoadReportFromSQL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Report From SQL</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>SQL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtSQL"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtTitle"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="optLandscape">
       <property name="text">
        <string>Landscape</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="optPortrait">
       <property name="text">
        <string>Portrait</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkShowLines">
       <property name="text">
        <string>Show lines between rows</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkShowPageNumbers">
       <property name="text">
        <string>Show Page Numbers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="cmdPrintOnly">
       <property name="text">
        <string>Print Only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdPrintAndSave">
       <property name="text">
        <string>Print and Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdExit</sender>
   <signal>clicked()</signal>
   <receiver>LoadReportFromSQL</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
